app/javascript/src/sol_prices_charts.js

```diff
@@ -4,9 +4,101 @@ import { DateTime } from "luxon";
 import 'chartjs-adapter-luxon';
 import '../lib/chart-financial';
 
+const coinGeckoString = 'exchange=coin_gecko'
+const ftxString = 'exchange=ftx'
+
 document.addEventListener('turbolinks:load', () => {
-  const chartCoinGecko = document.getElementById('myChart')
-  const chartFtx = document.getElementById('myChart2')
+  window.drawChart = drawChart;
+
+  drawChart();
+  
+  // Add event listeners to tab buttons.
+  const chartTabButtons = document.getElementsByClassName('chartTabButton');
```

Comment:
I think that `chartTabButton` is too generic. Maybe try to add `solPrices` to the class name.